src/utilities/utility.ts

```diff
@@ -129,6 +138,54 @@ export function annotateChildPods(kubectl: Kubectl, resourceType: string, resour
     return commandExecutionResults;
 }
 
+export async function getDeploymentConfig(): Promise<DeploymentConfig> {
+
+    let helmChartPaths = (process.env.HELM_CHART_PATHS && process.env.HELM_CHART_PATHS.split('\n').filter(path => path != "")) || [];
+    helmChartPaths.map(helmchart => {
+        let pathValue = helmchart;
+        if (!isHttpUrl(helmchart)) {  //if it is not an http url then convert to link from current repo and commit
+            let pathLink: string = `https://github.com/${process.env.GITHUB_REPOSITORY}/blob/${process.env.GITHUB_SHA}/${helmchart}`;
+            pathValue = pathLink;
+        }
+        return pathValue;
+    });
+
+    let inputManifestFiles: string[] = [];
+    if (!helmChartPaths || helmChartPaths.length == 0) {
+        let inputManifestFiles = inputParams.manifests || [];
+        inputManifestFiles.map(manifestFile => {
+            let pathValue = manifestFile;

```

Comment:
Move this logic to a utility method and reuse for all the cases (Helm, Manifest, Docker).
Something like - `getNormalizedPath(path)` that checks if it is already a URL or not, and returns the URL.

Reply:
Done